shuffle/cli/src/shared.rs

```diff
@@ -62,17 +62,23 @@ pub fn read_config(project_path: &Path) -> Result<Config> {
 }
 
 /// Send a transaction to the blockchain through the blocking client.
-pub fn send(client: &BlockingClient, tx: diem_types::transaction::SignedTransaction) -> Result<()> {
+pub fn send_transaction(
+    client: &BlockingClient,
+    tx: diem_types::transaction::SignedTransaction,
+) -> Result<()> {
     use diem_json_rpc_types::views::VMStatusView;
 
     client.submit(&tx)?;
-    assert_eq!(
-        client
-            .wait_for_signed_transaction(&tx, Some(std::time::Duration::from_secs(60)), None)?
-            .into_inner()
-            .vm_status,
-        VMStatusView::Executed,
-    );
+    let status = client
+        .wait_for_signed_transaction(&tx, Some(std::time::Duration::from_secs(60)), None)?
+        .into_inner()
+        .vm_status;
+    if status != VMStatusView::Executed {
+        return Err(anyhow::anyhow!(
+            "Unable to send transaction from client, got status: {}",
```

Comment:
The txn was submitted successfully, but the execution is failed.
 So I think something like "Transaction execute failed: {vm_status}"  maybe better error message.